src/react/workflow/ConfigurationTab.tsx

```diff
@@ -64,65 +58,101 @@ const ConfigurationHeader = styled.div`
 
 function ConfigurationTab({
   wfSelected,
-  replications,
   bucketList,
   locations,
 }: Props) {
   const dispatch = useDispatch();
   const history = useHistory();
   const queryClient = useQueryClient();
   const { workflowId } = wfSelected;
-  const isDeleteModalOpen = useSelector(
-    (state: AppState) => state.uiWorkflows.showWorkflowDeleteModal,
-  );
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+  const mgnt = useManagementClient();
+  const state = useSelector((state: AppState) => state);
+  const { instanceId } = getClients(state);
+  const accountId = getAccountId(state);
+
+  const replicationsQuery = useQuery({
```

Comment:
Can we move this query to queries.ts in order to refactor and avoid duplication in ConfigurationTab.tsx and ReplicationForm.tsx?

Reply:
The query is already mutualised we are using various selectors in 3 different places ConfigurationTab, ReplicationForm and Workflows. It may make sense to factorise this selector through `(workflows) => workflows.filter(w => w.replication).map(w => w.replication)` I will check that with the next lifecycle implem